instaloader/__main__.py

```diff
@@ -66,6 +71,46 @@ def filterfunc(item) -> bool:
 
     return filterfunc
 
+def get_cookies_from_instagram(domain, browser, cookiefile="", cookieName=''):
+    supported_browsers = {
+        "chrome": browser_cookie3.chrome,
+        "firefox": browser_cookie3.firefox,
+        "edge": browser_cookie3.edge,
+        "brave": browser_cookie3.brave,
+        "opera": browser_cookie3.opera,
+        "safari": browser_cookie3.safari
+    }
+
+    if browser not in supported_browsers:
+        print("Loading cookies from the specified browser failed")
+        print("Supported browsers are Chrome, Firefox, Edge, Brave, Opera and Safari")
+        return {}
+
+    cookies = {}
+    browser_cookies = list(supported_browsers[browser](cookie_file=cookiefile))
```

Comment:
It would be nice if the parameter names were more consistent in their style, e.g. `cookie_file` and `cookie_name` rather than mixing `cookieName` and `cookiefile`, `cookie_file`.

Reply:
Done.